src/test/java/Homework23.java

```diff
@@ -0,0 +1,36 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework23 extends BaseTest{
+
+    @Test
+
+public void deletePlaylist() {
+       String  playlistDeletedMsg = "Deleted playlist\"New playlist\"";
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.provideEmail("tassadit.talbi@testpro.io").providePassword("Password2026@").clickSubmit();
+        homePage.clickPlaylist().clickDeleteBtn();
+        Assert.assertEquals(homePage.getDeletedPlaylistMsg(),playlistDeletedMsg);
+
+
+
+
+
+
+
+    }
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._